extensions/nino-plus/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "nino-plus",
+  "title": "nino+",
+  "description": "nino+の関連リソースにアクセスできます",
+  "icon": "command-icon.png",
+  "author": "dninomiya",
+  "categories": [
+    "Web"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "lofi",
+      "title": "Lofi",
+      "description": "Lofi with ポモドーロで作業に集中",
+      "mode": "no-view"
+    },
+    {
+      "name": "discussions",
+      "title": "Discussions | GitHub",
+      "description": "質問やリクエストを行います",
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension. If the locale might affect functionality (e.g. using the correct unit of measurement), please use the preferences API.